frontend/src/modules/Emailing/Components/Emailing.tsx

```diff
@@ -135,24 +135,24 @@ export const Emailing = () => {
   )
 }
 
-/* Returns (number)
-    SUCC -> 201 
-    FAIL -> 401  */
-export const sendEmail = async (
-  emailItems: any,
-  setEmailSent: (arg: boolean) => void
-) => {
+/** Returns (bool)
+    SUCC -> true 
+    FAIL -> false  */
+export const sendEmail = async (emailItems: any) => {
```

Comment:
Might be more idiomatic to throw errors, especially because this is an async function so you can do things like
```js
sendEmail(emailItems)
  .then(/* something */)
  .catch(error => /* handle email failure */)
```
Or maybe not because failure to send is expected to happen sometimes

Would like other people's input here @hexbabe @CollinWoo @mluo24 (look at rest of PR for more context)